ImperatorToCk2/src/main/java/ImperatorToCK2/Main.java

```diff
@@ -1,689 +1,754 @@
-package ImperatorToCK2;
-
-import java.util.Scanner;
-import java.util.ArrayList;
-import java.io.File;
-import java.io.IOException;
-import java.io.FileInputStream;
-import java.io.PrintWriter;
-import java.io.StringWriter;
-import java.io.FileOutputStream;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-
-import ImperatorToCK2.ConverterLogger;

```

Comment:
You probably need this import.

Reply:
`import ImperatorToCK2.ConverterLogger` actually causes an error, as long as the .java files are there in the program, import statements don't need to be added